_profiles/v2-article.html

```diff
@@ -38,6 +38,10 @@
 #
 budgets:
   -
+    metric: TTFB
```

Comment:
The reason I didn't go for this in the first place is it's a bit of a userless metric.
[There's a bit of a blog here about it](https://blog.cloudflare.com/ttfb-time-to-first-byte-considered-meaningles/).